packages/docs/docs/tutorial/integrate-sdk/android.mdx

```diff
@@ -108,22 +97,22 @@ import io.logto.sdk.android.LogtoClient;
 import io.logto.sdk.android.type.LogtoConfig;
 
 public class MainActivity extends AppCompatActivity {
-		private LogtoClient logtoClient;
+        private LogtoClient logtoClient;
 
-		@Override
+        @Override
         protected void onCreate(Bundle savedInstanceState) {
             super.onCreate(savedInstanceState);
             setContentView(R.layout.activity_main);
 
-			LogtoConfig logtoConfig = new LogtoConfig(
+            LogtoConfig logtoConfig = new LogtoConfig(
                 "<your-logto-endpoint>",
                 "<your-app-id>",
                 null,
                 null,
                 true
-		    );
-	
-			logtoClient = new LogtoClient(logtoConfig, getApplication());
+            );
+    
+            logtoClient = new LogtoClient(logtoConfig, getApplication());
```

Comment:
It seems like we should decrease the indent by 4 spaces in this fragment of code